spec/cucumber/formatter/legacy_api/adapter_spec.rb

```diff
@@ -1703,7 +1703,7 @@ def apply_before_hooks(test_case)
             end
 
             expect( formatter.legacy_messages ).to eq([
-                :before_features,
+                                                        :before_features,
                   :before_feature,
```

Comment:
I wouldn't change the indentation of this file. I think the spaces gives an intuition of what is going on and following the rules here has less value. What do you think @phoebeclarke @mattwynne ?